locales/fr/fr.json

```diff
@@ -217,7 +217,7 @@
     "Faroe Islands": "Îles Féroé",
     "February": "Février",
     "Fiji": "Fidji",
-    "Finish enabling two factor authentication.": "Finish enabling two factor authentication.",
+    "Finish enabling two factor authentication.": "Terminez l'activation de la double authentification",
```

Comment:
Add a full stop at the end of a sentence.

Reply:
je plussoie, un petit point et puis c'est bon